src/chrome/komodo/content/hyperlinks/filehandler.js

```diff
@@ -210,6 +210,35 @@
     // Limit to JavaScript string styles.
     ko.hyperlinks.handlers.jsGetElementByIdHandler.limitToTheseStyles([Components.interfaces.ISciMoz.SCE_UDL_CSL_STRING]);
 
+    /**
+     * A Smarty include file handler - to help in opening files. Examples:
+     *    {include file="path/to/file.tpl"}
+     */
+    function smarty_jump_handler(match) {
+        var filepath = match[2];
+
+        // If not absolute path, see if there's a "/templates/" parent of current file.
+        if (filepath[0] != "/")
+        {
+            var view_filepath = ko.views.manager.currentView.koDoc.file.path;
+            var smartyTemplatePathMatch = /^(.*[\/\\]templates[\/\\])/.exec(view_filepath);
+            if (smartyTemplatePathMatch !== null)
+                filepath = smartyTemplatePathMatch[1] + filepath;
+        }
+
+        // Open file
+        filename_jump_handler(filepath);
+    }
+    ko.hyperlinks.handlers.smartyIncludeHandler =
+        new ko.hyperlinks.RegexHandler(
+            "Smarty include handler",
+            new RegExp("{include\\s+(.*\\s)?file=[\"']([^'\"}]+)[\"']", "i"),
```

Comment:
I assume you've tested this so you know it works but why is `\\s` working in this context?  I tried it with the example you provided in the Komodo regex editor and it doesn't work.  If I remove the first `\` then it works fine.  If you've tested it and it works the it's fine with me, bonus points if you tell me why though!

Reply:
Yes, it works with double slash and not with a single.  I think the reason is that we are in a Javascript string, so `\s` becomes `s` meaning regex searches for an "s."  `\\s` becomes `\s`, which regex searches for whitespace.

Reply:
@ssigwart Sorry for the delay!  Right, you jogged my memory.  You are right about it being a string in JS.  Can you wrap it in backticks and see if that fixes the issues?

Reply:
So this
```javascript
new RegExp(`{include\\s+(.*\\s)?file=[\"']([^'\"}]+)[\"']`, "i")
```

Reply:
Nvm it doesn't work.